python/tests/BUILD

```diff
@@ -0,0 +1,13 @@
+cc_test(
+    name = "cc_deps_link_test",
+    size = "small",
+    srcs = [
+        "cc_deps_link_test.cc",
+    ],
+    copts = [
+        "-std=c++14",
+    ],
```

Comment:
let's make a bug and add a `TODO(b/)` to see if this needs to be made more accessible to other targets in the repo. And give reason for why it is needed.

This may be useful as we think about generalizing:
https://github.com/tensorflow/tflite-micro/blob/836a4ad1bc14d94ba10ac289ffe5fb797d8d0fbe/tensorflow/lite/micro/build_def.bzl#L1-L7

Reply:
Added in 470fffb.